setup.py

```diff
@@ -350,9 +338,10 @@ def prepare_cew_for_windows():
     license=PKG_LICENSE,
     keywords=PKG_KEYWORDS,
     classifiers=PKG_CLASSIFIERS,
+    cmdclass={'build_ext': build_ext},
```

Comment:
Create a ``CMDCLASS`` constant above, assign it the dictionary, and use it here in the ``setup()`` call.